config-service-impl/src/main/java/org/hypertrace/config/service/ConfigServiceGrpcImpl.java

```diff
@@ -0,0 +1,139 @@
+package org.hypertrace.config.service;
+
+import com.google.protobuf.Struct;
+import com.google.protobuf.Value;
+import io.grpc.stub.StreamObserver;
+import java.util.LinkedHashMap;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Given that we're already using guava in this project as part of grpc, is there a compelling reason to bring in a second util lib?

Reply:
removed the Apache commons lib